### PR TITLE
Updated theme to the latest version

### PR DESCRIPTION
**Pull request by fadeev**

### CODE REVIEW

File: hellochain/tutorial/03-types.md [hunk 1]
```diff
@@ -10,4 +10,5 @@ define this struct and other necessary components in a types file.
 
 Save this to `x/greeter/internal/types/types.go`
 
-<<< @/hellochain/x/greeter/internal/types/types.go
++++ https://github.com/cosmos/sdk-tutorials/blob/master/hellochain/x/greeter/internal/types/types.go
```
Comment (hschoenburg) [inline hellochain/tutorial/03-types.md]: hmm relative paths dont work anymore? we have to use URLs? why is that if you dont mind me asking

File: hellochain/tutorial/15-full-app.md [hunk 2]
```diff
@@ -11,4 +11,4 @@ and add greeter's AppModuleBasic and AppModule alongside the other modules.
 
 Update your `app.go` to look like the following
 
-<<< @/hellochain/app.go{12,25,26,33,36,39,44,45,49,50,53}
```
Comment (hschoenburg) [inline hellochain/tutorial/15-full-app.md]: is there anyway we can keep the line highlights? they are pretty useful

File: hellochain/tutorial/03-types.md (hunk 1 above)
Comment (fadeev) [reply hellochain/tutorial/03-types.md]: We didn't use relative paths in the SDK docs, so this feature got kind of forgotten. I've added it back and modified MDs accordingly.

File: hellochain/tutorial/15-full-app.md (hunk 2 above)
Comment (fadeev) [reply hellochain/tutorial/15-full-app.md]: We're using `{...}` syntax for annotating things like synopsis and prerequisite links, so we can't use it for line numbers. I'm thinking of using `.../app.go[12,25,26]` syntax instead.

**Pull request merged**